third_party/move/move-model/src/metadata.rs

```diff
@@ -15,20 +15,22 @@ use std::{
 };
 
 const UNSTABLE_MARKER: &str = "-unstable";
-// 2.2, even though stable, produces several warnings in the frameworks which we first need to fix
-// before we can make it the default
-pub const LATEST_STABLE_LANGUAGE_VERSION_VALUE: LanguageVersion = LanguageVersion::V2_2;
-pub const LATEST_STABLE_COMPILER_VERSION_VALUE: CompilerVersion = CompilerVersion::V2_0;
+
```

Comment:
We may want to say what "stable" means (tested to be reliable on mainnet, or something similar here) as opposed to "latest".